configure.ac

```diff
@@ -179,7 +179,6 @@ AC_ARG_ENABLE([jemalloc],
 	  else
 	     jemalloc_enabled=no
 	  fi])
-        #[jemalloc_enabled=yes])
```

Comment:
Please add some comment here - jemalloc disabled by default since we use build in ptmalloc3